sw/legacy/demo/ethernet/ksz8851.c

```diff
@@ -222,7 +222,7 @@ static void ksz8851_recv(struct netif *netif) {
 
     // Reset QMU RXQ frame pointer to zero.
     ksz8851_reg_write(spi, ETH_RXFDPR, 0x4000);
-    
+
```

Comment:
Is this a tab to space change? (Or vice versa hard to tell in the github UI...)

Reply:
This removes trailing whitespace.

Reply:
Moved this to a cleanup commit with message saying what's done.